plugins/extra/watchdog.go

```diff
@@ -0,0 +1,76 @@
+package extra
+
+import (
+	"time"
+
+	"github.com/go-xorm/xorm"
+
+	seabird "github.com/belak/go-seabird"
+)
+
+func init() {
+	seabird.RegisterPlugin("watchdog", newWatchdogPlugin)
+}
+
+type watchdogPlugin struct {
+	db *xorm.Engine
+}
+
+type watchdogCheck struct {
+	Time   time.Time `xorm:"created"`
```

Comment:
Just noticed, there's no PK here?

Reply:
There isn't. The idea is that None of the fields need to be unique, and technically you could check multiple times in the same second if you wanted.